source/macros.tex

```diff
@@ -317,13 +318,13 @@
 
 %% Miscellaneous
 \newcommand{\uniquens}{\placeholdernc{unique}}
-\newcommand{\stage}[1]{\item{\textbf{Stage #1:}}\xspace}
-\newcommand{\doccite}[1]{\textit{#1}\xspace}
+\newcommand{\stage}[1]{\item{\textbf{Stage #1:}}}
+\newcommand{\doccite}[1]{\textit{#1}}
 \newcommand{\cvqual}[1]{\textit{#1}}
 \newcommand{\cv}{\cvqual{cv}}
-\renewcommand{\emph}[1]{\textit{#1}\xspace}
-\newcommand{\numconst}[1]{\textsl{#1}\xspace}
-\newcommand{\logop}[1]{{\footnotesize #1}\xspace}
+\renewcommand{\emph}[1]{\textit{#1}}
```

Comment:
Separate issue: If you just want `\emph` to have the same effect as `\textit` (_i.e._, the text is always italic regardless of the context), then you could just write `\let\emph\textit`.

Reply:
Isn't that the default definition of `\emph` anyway?

Reply:
`\emph` alternates between italics and roman type as its nested, while `\textit` does not.

For example, `\emph{Hello, \emph{world}!}` would be typeset as “<i>Hello,</i> world<i>!</i>” In contrast, `\textit{Hello, \textit{world}!}` would be typeset as “<i>Hello, world!</i>”

Reply:
Ah yes, of course. Luckily we aren't quite that nestedly empathetic in the standard. :-)

Reply:
In fact, it's shocking how little we \emphasize in the standard compared to the time we spend talking about it!